db/models.py

```diff
@@ -1,4 +1,12 @@
+from enum import unique
```

Comment:
There is a syntax error in the import statement. It should be `from enum import Enum` if you intend to use the `Enum` class from the `enum` module.